### PR TITLE
Fix log service interface

### PR DESCRIPTION
**Pull request by mmateja**
After few recent major changes LogService interface became incompatible
with PagerDutyService. I'm fixing it in this PR.

### CODE REVIEW

File: spec/nine_one_one/pager_duty_service_spec.rb [hunk 1]
```diff
@@ -13,15 +13,16 @@
       use_vcr_cassette 'pager_duty_trigger_event_success', match_requests_on: [:body]
 
       it 'calls pager duty service with proper json' do
-        pager_service.trigger_event('Incident description', source: 'source info', details_hash: details_hash, severity: severity)
+        pager_service.trigger_event('Incident description', source: 'source info', details_hash: details_hash,
```
Comment (przemeq84) [inline spec/nine_one_one/pager_duty_service_spec.rb]: If you started using `message` I'd get rid of the `'Incident description'` here as well.

**Review by przemeq84 — APPROVED** [review]

**Comment by przemeq84** [comment]
_green hat_:
I still believe this library could use a generic `call(accident, details_hash)` method common in all services (`LogService`, `PagerDutyService` and `SlackService`). Having this we could control service type <-> severity level mapping in the configuration file.

**Comment by przemeq84** [comment]
And let @Ciebiada know about this change - he was planning to fix it by himself.

**Review by Ciebiada — APPROVED** [review]
very not bad

**Comment by madlew** [comment]
it's actually not a bug fix so I'd change the version to '2.1.0' than '2.0.4' but it's a minor thing

**Review by madlew — APPROVED** [review]

**Comment by mmateja** [comment]
@madlew it's definitely a bug. There is an assumption that `PagerDutyService` and `LogService` implement the same interface - they are used interchangeably (https://github.com/u2i/nine_one_one/blob/master/lib/nine_one_one/notifier.rb#L17). This was broken with some major changes in the past.

**Pull request merged**